Classes/NSManagedObject+ActiveRecord.m

```diff
@@ -366,17 +366,23 @@ - (void)setSafeValue:(id)value forKey:(NSString *)key {
 
     else if ([value isKindOfClass:[NSString class]]) {
 
-        if ([self isIntegerAttributeType:attributeType])
```

Comment:
What if we keep this, but change `numberWithInteger`, below to be `numberWithLongLong`. That should satisfy 32-bit systems :+1: